config.yaml

```diff
@@ -63,38 +63,25 @@ wiki:
         projects:
             title: Projekt <YEAR>
             template: Mall:Årsida för projekt
-        program_overview:
-            colours:
-                - "#339966"
-                - "#006699"
-                - "#990000"
-                - "#FF8C00"
-            title: Verksamhetsplan <YEAR>/Måluppfyllnad/Programöversikt
-            templates:
-                page: Mall:Programöversikt
-                program: Mall:Programöversikt/Program
-                strategy: Mall:Programöversikt/Strategi
-                goal: Mall:Programöversikt/Mål
-                project: Mall:Programöversikt/Projekt
         categories:
             general: Föreningen <YEAR>
             pages:
                 Föreningen <YEAR>: Föreningen
                 Projekt <YEAR>: Projekt efter verksamhetsår
                 Ledningsrapporter <YEAR>: Ledningsrapporter
                 Ansökningar <YEAR>: Ansökningar
+                Omnämnande <YEAR>:
+                    - Omnämnande
+                    - Media
                 Resor <YEAR>: Resor
                 Årsmöte <YEAR>: Årsmöte
                 Checklistor <YEAR>: Checklistor
```

Comment:
Would we not expect this,  or a replacement,  in 2024? Its included on "Mall:Metrics-sida"

Reply:
I don't understand what you mean. From https://phabricator.wikimedia.org/T336757 it sounds like we should not add these.

Reply:
I think the comment somehow got attached to the wrong row (looks right in file view but not on the PR view).

My comment is about the removal of `Globala mätetal <YEAR>: Globala mätetal`

Reply:
Should it still be called "Globala mätetal" or should it also change name?

Reply:
I think we can keep the name. It's an internal category and it works for both Global Metrics and Core metrics (they are both types of global metrics)

Reply:
I re-added it in the last commit since the removal did not sound intentional